recipe/build.sh

```diff
@@ -1,3 +1,4 @@
 # use bootstrapped pip to install certifi without depending on installed pip
+ls -1 "${BUILD_PREFIX}/lib"
```

Comment:
So [on CI]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=483048&view=logs&jobId=517fe804-fa30-5dc2-1413-330699242c05&j=517fe804-fa30-5dc2-1413-330699242c05&t=c10fa5f2-fdf6-5338-3bdb-c4bea7c23412 ) this directory appears not to exist. Would think the packages in `build` would have added things here. However I may very well be missing things about how cross-python builds work.